dev/test/collection.ts

```diff
@@ -211,6 +211,68 @@ describe('Collection interface', () => {
     });
   });
 
+  it('for CollectionReference.withConverter().add()', async () => {
+    let docId = '';
+    const overrides: ApiOverride = {
+      commit: request => {
+        docId = request.writes![0].update!.name!;
+        // Remove the auto-generated id and then verify that the rest of the
+        // protobuf matches.
+        delete request.writes![0].update!.name;
```

Comment:
Instead of deleting the field, can you update the expected request to include the extracted docId? Might be slightly cleaner.

Reply:
done.